kdb-set: add line about negative values to usage text

kodebach opened the pull request:
# Purpose

Update the usage text of `kdb set` to indicate how to set negative values.

# Checklist
- [ ] commit messages are fine ("module: short statement" syntax and refer to issues)
- [ ] I added unit tests
- [ ] I ran all tests locally and everything went fine
- [ ] affected documentation is fixed
- [ ] I added code comments, logging, and assertions (see doc/CODING.md)
- [ ] meta data is updated (e.g. README.md of plugins)
- [ ] release notes are updated (doc/news/_preparation_next_release.md)

@markus2330 please review my pull request

markus2330 commented:
Thank you!

Please update "doc/help/kdb-set.md" (or even better doc/help/kdb.md). These are the man pages. I think the text you are updated is not displayed anymore.

And please add a line to the release notes.

kodebach commented:
@markus2330 I updated the other documentation as well. The text I updated is displayed (at least for me) for example if you try to execute `kdb set /tests/neg -3`.

markus2330 commented on doc/help/kdb-set.md:
```diff
@@ -59,6 +65,9 @@ To create a new key with a null value:
 To set a key to an empty value:
 `kdb set user/example/key ""`
 
+To set a key to a negative value:
```
<br> is needed to actually have a newline. (other examples are wrong, too)

markus2330 commented on doc/help/kdb-set.md:
```diff
@@ -16,6 +16,10 @@ This command allows the user to set the value of an individual key.
 
+## NEGATIVE VALUES
+
+To set a key to a negative value, `--` has to be used to stop option processing. (example below)
```
see example below

markus2330 commented on doc/news/_preparation_next_release.md:
```diff
@@ -125,6 +125,7 @@ We improved the documentation in the following ways:
   - [`uname`](https://www.libelektra.org/plugins/uname),
   plugin. *(René Schwaiger)*
 - We improved the formatting of our [compilation guide](/doc/COMPILE.md). *(René Schwaiger)*
+- The documentation for `kdb`and `kdb set` now mention the `--` option to stop option processing. This is useful for setting negative values among other things.
```
Please add your name. (See build server error)

markus2330 commented:
Please rebase and add your name in the release notes.

kodebach commented:
@markus2330 should be ready to merge

markus2330 commented:
Thank you, great job!
